packages/repository/src/decorators/model.ts

```diff
@@ -81,14 +81,28 @@ export function property(definition?: Partial<PropertyDefinition>) {
 export namespace property {
   export const ERR_PROP_NOT_ARRAY =
     '@property.array can only decorate array properties!';
+  export const ERR_NO_ARGS = 'Should have at least one argument';
+  export const ERR_PROP_IS_ARRAY = 'Cannot use @property.union on arrays';
```

Comment:
The code (API) is using `oneOf`, this error message says `union`. Please make it consistent. (I think this is just an oversight on your part?)